src/model/gateway.rs

```diff
@@ -453,93 +458,87 @@ bitflags! {
         ///
         /// **Info**: This intent is *privileged*. In order to use it, you must head to your
         /// application in the Developer Portal and enable the toggle for *Privileged Intents*.
-        ///
-        /// This intent is also necessary to even receive the events in contains.
         const GUILD_MEMBERS = 1 << 1;
-        /// Enables following gateway events:
+        /// Enables the following gateway events:
         /// - GUILD_AUDIT_LOG_ENTRY_CREATE
         /// - GUILD_BAN_ADD
         /// - GUILD_BAN_REMOVE
         const GUILD_MODERATION = 1 << 2;
-        /// Backwards compatibility with old gateway event name. Same as GUILD_MODERATION
-        const GUILD_BANS = 1 << 2;
```

Comment:
After some thought, I think the best course of action is to just keep the old name and mark it with ```#[deprecated = "use `GatewayIntents::GUILD_MODERATION` instead"]```.

Reply:
Alright, will do in a bit